docs/porting/libraries.md

```diff
@@ -1,5 +1,304 @@
+
+It's recommended that you pick a single .NET Platform Standard version and use that throughout your library.
+
+Read more in [The .NET Platform Standard](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md).
+
+## .NET Core project model overview
+
+.NET Core introduces a new project model that you may wish to spend some time understanding before you begin porting your code.  Rather than provide a comparison with the .NET Framework project model, this section covers the essential pieces.
+
+### The project file: `project.json`
+
+.NET Core projects are defined by a directory containing a `project.json` file.  This file is where aspects of the project are declared, such as package dependencies, compiler configuration, runtime configuration, and more.
+
+The `dotnet restore` command reads this project file, restores all dependencies of the project, and generates a `project.lock.json` file.  This file contains all the necessary information the build system needs to build the project.
+
+To learn more about the `project.json` file, read the [.NET Core project system docs]((../project-model/overview.md).
+
+### The solution file: `global.json`
+
+The `global.json` file is an optional file to include in a solution which contains multiple projects.  It typically resides in the root directory of a set of projects.  It can be used to inform the build system of different subdirectories which can contain projects.  This is for larger systems composed of several projects.
+
+To learn more about the `global.json` file, check out the [.NET Core project system docs](../project-model/global-json-reference.md).
+
+## How to Multitarget with .NET Core
+
+Many libraries multitarget to have as wide of a reach as possible.  With .NET Core, multitargeting is a "first class citizen", meaning that you can easily generate platform-specific assemblies.
+
+Multitargeting is as simple as adding the correct Target Framework Moniker (TFM) to your `project.json` file, pulling in the correct dependecies (`dependencies` for .NET Core and `frameworkAssemblies` for .NET Framework), and potentially using `#if` directives to conditionally compile source.
+
+For example, imagine you were building a library where you wanted to perform some networking, and you wanted that library to run on all .NET Framework versions, a Portable Class Library (PCL) Profile, and .NET Core.  For .NET Core and .NET Framework 4.5+ targets, you may use `System.Net.Http` and `async`/`await`.  However, for earlier versions of .NET Framework, those APIs aren't available.
+
+Here's a sample `project.json` for that scenario targeting .NET Framework versions 2.0, 3.5, 4.0, 4.5, and .NET Platform Standard 1.5:
```

Comment:
I'd rather not.  The concept of properly targeting older versions is often one of the most challenging things to figure out how to do.  I'd like to bring that up the forefront here, given the breadth of support many libraries require.